lib/commentOnClosedIssue.js

```diff
@@ -116,4 +116,4 @@ function statusCodeIsOk(response) {
     return false;
 }
 
-commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com[^\s.,:]*/ig;
+commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com[^\s.,:)\\]*/ig;
```

Comment:
Ahh, okay that makes sense, we definitely want to capture it so we can provide a list.  Thanks.